### PR TITLE
[WIP] Format Lookalike view

### PR DESCRIPTION
**Pull request by ajdapretnar**
View in Lookalike's TableView object is broken. Numbers are formatted incorrectly (should not be elided) and the table should be sorted in a descending order.

### CODE REVIEW

**Comment by ajdapretnar** [comment]
@PrimozGodec I wasn't very successful at this. I cannot figure out how to sort by column. I tried `self.neighbors_view.sortByColumn(1, 1)` but then I get an error I don't understand: `RuntimeError: Can't modify PyTableModel when it's sorted`. I don't know where I am modifying the table after sorting...
Probably @ales-erjavec knows how to elegantly set the decimal places in QTableView and how to sort by column. ⭐️

File: orangecontrib/lookalike/widgets/owlookalike.py [hunk 1]
```diff
@@ -333,11 +337,19 @@ def apply(self):
...
+        index_0 = self.neighbors_model.index(index, 0)
+        index_1 = self.neighbors_model.index(index, 1)
+
+        name = index_0.data(Qt.DisplayRole)
+        distance = index_1.data(Qt.EditRole)
+        if isinstance(distance, numbers.Real) and not math.isnan(distance):
+            numtext = "{:.6g}".format(distance)
+            text = "I am {} (unspecified units) away from {}".format(numtext, name)
+        else:
+            text = "I am not a number!"  # I am a free man!
```
Comment (ajdapretnar) [inline orangecontrib/lookalike/widgets/owlookalike.py]: 🥇

File: orangecontrib/lookalike/widgets/owlookalike.py [hunk 2]
```diff
@@ -330,10 +337,19 @@ def apply(self):
                           img_negh, self.orange_logo_item])
         widget.setPos(0, 60)
         self.scene.addItem(widget)
-
-        title = QGraphicsSimpleTextItem("I am {:.2f}% {}".format(
-            float(self.neighbors_model.index(self.neighbor_index, 1).data()),
```
Comment (ales-erjavec) [inline orangecontrib/lookalike/widgets/owlookalike.py:335]: The distances cannot in general be used as a 'ratio' of similarity as was done here. You might (kind of) do that for cosine, pearson, abs pearson, ...) using `1 - distance`.

Comment (ajdapretnar) [reply orangecontrib/lookalike/widgets/owlookalike.py:335]: We were aware of that. This was meant solely for demonstration purposes where we showed the Celebrity Lookalike project to kids and they usually wanted to know 'how similar they are to a person'. This is impossible to quantify this way, so we cheated and created a false measure of similarity. Any other ideas welcome.
(p.s. We would be using this widget with cosine distance anyway...)

Comment (PrimozGodec) [reply orangecontrib/lookalike/widgets/owlookalike.py:335]: @ajdapretnar @ales-erjavec Wanted to say that current implementation of the Lookalike widget changes the distance to percents. That means that we are showing how much different two persons are and not how similar they are. If we would want to tell you are so many percents similar to that person we would need to transform distance to similarity with `1-distance`. Currently, bigger value means that persons are more different and smaller than they are more similar. I think that Neighbor widget changed in this aspect from the last year.

**Pull request merged**